daemon/src/model/cfd.rs

```diff
@@ -805,6 +851,51 @@ impl Cfd {
                     self.state
                 ),
             },
+            CfdStateChangeEvent::ProposalSigned(transaction) => match self.state.clone() {
+                CfdState::Open {
+                    common,
+                    dlc,
+                    attestation,
+                    ..
+                } => CfdState::Open {
+                    common,
+                    dlc,
+                    attestation,
+                    collaborative_close: Some((transaction, SystemTime::now())),
+                },
+                CfdState::PendingCommit {
+                    common,
+                    dlc,
+                    attestation,
+                    ..
+                } => CfdState::PendingCommit {
+                    common,
+                    dlc,
+                    attestation,
+                    collaborative_close: Some((transaction, SystemTime::now())),
+                },
+                _ => bail!(
+                    "Cannot add proposed settlement details to state because of unexpected state {}",
+                    self.state
+                ),
+            },
+            CfdStateChangeEvent::CloseSent => match self.state.clone() {
+                CfdState::Open {
+                    common,
+                    dlc,
+                    attestation,
+                    collaborative_close: Some(collaborative_close),
+                } => CfdState::PendingClose {
+                    common,
+                    dlc,
+                    attestation,
+                    collaborative_close,
+                },
+                _ => bail!(
+                    "Cannot transition to PendingCet because of unexpected state {}",
```

Comment:
:wrench: Little copy-paste error.

```suggestion
                    "Cannot transition to PendingClose because of unexpected state {}",
```